README.md

```diff
@@ -165,10 +166,11 @@ This is to follow the current upload status to the `Geostore` for a particular `
 
 It gives you the information for one or all the datasets created on the `Geostore`.
 
-| Argument              |            Description            |
-| --------------------- | :-------------------------------: |
-| `-s`, `--survey` TEXT |       The survey to filter        |
-| `-v`, `--verbose`     | Use verbose to display debug logs |
+| Argument                |                                Description                                |
+| ----------------------- | :-----------------------------------------------------------------------: |
+| `-s`, `--survey` TEXT   |                           The survey to filter                            |
+| `-d`, `--datatype` TEXT | Datatype _Only `imagery.historic` is available at the moment._ [required] |
```

Comment:
To keep the documentation aligned with the CLI help, you can use `poetry run [command] --help` and copy paste the return in the documentation.